gcn/notices/swift/bat/guano/alert.schema.json

```diff
@@ -1,26 +1,40 @@
 {
   "$id": "https://gcn.nasa.gov/schema/gcn/notices/swift/bat/guano/alert.schema.json",
   "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "Swift/BAT-GUANO Alert",
+  "description": "Candidate gamma-ray transient reported from the BAT-GUANO targeted search",
   "type": "object",
-  "properties": {
-    "alert_reporter": {
-      "$ref": "/schema/gcn/notices/core/Reporter.schema.json"
-    },
-    "alert_info": {
+  "anyOf": [
+    {
       "$ref": "/schema/gcn/notices/core/Alert.schema.json"
     },
-    "event": {
+    {
       "$ref": "/schema/gcn/notices/core/Event.schema.json"
     },
-    "datetime": {
+    {
       "$ref": "/schema/gcn/notices/core/DateTime.schema.json"
     },
-    "localization": {
+    {
       "$ref": "/schema/gcn/notices/core/Localization.schema.json"
     },
-    "statistics": {
+    {
       "$ref": "/schema/gcn/notices/core/Statistics.schema.json"
+    },
+    {
+      "$ref": "/schema/gcn/notices/core/Reporter.schema.json"
+    },
+    {
+      "$ref": "/schema/gcn/notices/core/AdditionalInfo.schema.json"
+    }
+  ],
+  "properties": {
+    "follow_up_event": {
```

Comment:
This field can be referred to core-schema at https://github.com/nasa-gcn/gcn-schema/blob/main/gcn/notices/core/Event.schema.json
Such that in example can be provided as:
"id": [bn....] or alternate
"id_source": [Fermi]

Reply:
The core schema you reference is inadequate because it does not communicate the event that triggered the followup